macros/StatusVmon904HV.py

```diff
@@ -0,0 +1,1339 @@
+#!/usr/bin/env python
+
+import cx_Oracle
+import ROOT
+import os
+import time
+from datetime import datetime
+from operator import itemgetter
+from array import array
+
+#in the DB the DeltaV between pins are saved, not the V from ground
+
+#time interval considered to extract the data
+#sta_period = "'2018-07-17 08:00:01'"
+#end_period = "'2018-07-17 14:30:01'"
+
+#to control the status there are data in the period 
+#sta_period = "'2018-12-06 22:00:01'"
+#end_period = "'2019-03-15 08:00:01'"
+
+#test with board 4 channel 2 : 2-2-Top
```

Comment:
Typically when committing code to the central repo we remove all commented lines.